60_networks.Rmd

```diff
@@ -0,0 +1,664 @@
+
+A typical workflow for estimating a microbial network involves several analysis steps, including methods for zero handling, data normalization, and computing microbial associations. The inferred association matrix is then sparsified and transformed into an adjacency matrix. The resulting network can be analyzed using local network properties, such as centrality measures, or global measures that describe the overall structure of the network. Network plots provide further insight into the microbial community structure and allow for an exploratory analysis of the microbial relationships.
+
+Since microbial interactions are likely to change between conditions, e.g., between healthy individuals and patients, identifying network differences between groups is often an integral secondary analysis step. Differences can be detected visually by plotting the two networks side by side, or quantitatively using differential network analysis tools.
+
+All of the above steps are implemented in the R package NetCoMi [@peschel2021netcomi], which provides functionality for the complete workflow of constructing, analyzing, and comparing networks for microbiome data. In this section, we demonstrate the use of NetCoMi.
+
+
+## Intruduction to NetCoMi
+
+The package has a modular structure with a function for each step of the workflow. These are the main functions with their purpose and expected input:
+
+| Function               | Purpose                | Input
+|:----------------------:|:----------------------:|:----------------------:|
+| `netConstruct`       | Network construction   | Count matrix, phyloseq object, or association matrix|
+| `netAnalyze`         | Network analysis       | Object of class `microNet` returned by `netConstruct`|
+| `plot.microNetProps` | Network plotting       | Object of class `microNetProps` returned by `netAnalyze`|
+| `netCompare`         | Network comparison     | Object of class `microNetProps` returned by `netAnalyze`|
+| `diffnet`            | Differential network construction | Object of class `microNet` returned by `netConstruct`|
+
+
+We start by installing NetCoMi from GitHub and loading the necessary packages.
+
+```{r install-pkg, eval=FALSE}
+if(!require(NetCoMi)){
+  devtools::install_github("stefpeschel/NetCoMi", force = TRUE,
+                         dependencies = c("Depends", "Imports", "LinkingTo"),
+                         repos = c("https://cloud.r-project.org/",
+                                   BiocManager::repositories()))
+}
+```
+
+```{r load_packages, message=FALSE, warning=FALSE}
+library(NetCoMi)
+library(mia)
+library(phyloseq)
```

Comment:
We do support translations between these two containers but this book does not support phyloseq, or phyloseq-based tools in general because this would lead to mixing of two rather different tool 
 ecosystems (TreeSE vs phyloseq) would be confusing from user perspective.